Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/PersonDetailsQrCard.kt

```diff
@@ -29,9 +31,15 @@ class PersonDetailsQrCard(parent: ViewGroup) :
         payloads: List<Any>
     ) -> Unit = { item, payloads ->
         val curItem = payloads.filterIsInstance<Item>().singleOrNull() ?: item
-        image.setImageBitmap(curItem.qrCodeBitmap)
+
         curItem.apply {
-            qrCodeBitmap?.let { progressBar.hide() }
+            image.loadAny(
+                CoilQrCode(content = curItem.certificate.qrCode)
+            ) {
+                crossfade(true)
+                placeHolderView(image, progressBar)
```

Comment:
Looks unusual for me , why do we have to specify the` image` and `progressBar` as placeHolder ?
should not be only the progress bar?

Reply:
From inside the Coil listener, afaict, we don't have access to the target view without guessing/casting).
So we need the reference to to show/hide it.

Reply:
The best solution would probably be to no longer have a "progressbar" but use an animated drawable that we set.
Outside of this PR's scope thought.